src/SimpleThings/EntityAudit/EventListener/CreateSchemaListener.php

```diff
@@ -109,7 +109,7 @@ public function postGenerateSchema(GenerateSchemaEventArgs $eventArgs)
             'autoincrement' => true,
         ));
         $revisionsTable->addColumn('timestamp', 'datetime');
-        $revisionsTable->addColumn('username', 'string');
+        $revisionsTable->addColumn('username', 'string')->setNotnull(false);
```

Comment:
oh in this case you are right, but changing a schema could be a problem (not in this case) if the schema change is not recognized and handled on production